src/app/shared/profile-info/profile-info.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit, Input, OnChanges } from '@angular/core';
+
+import { RequestService } from '../../RequestService/requests';
+import { CURRENT_YEAR, MEDIA_XS } from '../../config';
+
+@Component({
+  selector: 'profile-info',
+  templateUrl: './profile-info.component.html',
+  styleUrls: ['./profile-info.component.css']
+})
+export class ProfileInfoComponent implements OnChanges {
+  @Input() profileUserName: string;
+  @Input() showPhoto: boolean;
+  @Input() showEmail: boolean;
+  @Input() fields: string[];
+  profileData: any;
+  media_xs = MEDIA_XS;
+  MASK = 'https://aswwu.com/#/profile';
+
+  constructor( private request: RequestService ) { }
+
+  usernameFallback() {
```

Comment:
Great fallback. As I mentioned earlier, we could look into the possibility of also setting the `userProfile.photo` to be the Mask Unknown photo.